tests/slsa_analyzer/build_tool/__snapshots__/test_poetry.ambr

```diff
@@ -0,0 +1,14 @@
+# serializer version: 1
+# name: test_get_build_dirs[mock_repo0]
+  list([
+    PosixPath('.'),
+  ])
+# ---
+# name: test_get_build_dirs[mock_repo1]
+  list([
+  ])
+# ---
+# name: test_get_build_dirs[mock_repo2]
+  list([
```

Comment:
Shouldn't we detect a build here? Why is this list empty?

Reply:
Made changes in `defaults.ini` as well as `Poetry.get_build_dirs()` to address this issue ([9c167f2](https://github.com/oracle-samples/macaron/pull/67/commits/9c167f271c3b880c3db95d6de5fc7bb51bdfcf3d)).